Frameworks/CoreFoundation/Base.subproj/CFInternal.h

```diff
@@ -363,10 +359,9 @@ CF_PRIVATE Boolean __CFProcessIsRestricted();
 
 // WINOBJC: The reference platform uses a special compiler option not available to us to 
 // get __CFConstantStringClassReferencePtr filled out with the correct "Class". Since this isn't

```

Comment:
Can you propagate this comment to CFBridgeUtilities.h? It might be useful to pretend CFBridgeUtilities is an upstream header with our own modifications (for change-tracking purposes)